#!/usr/bin/env python
PACKAGE = "jsk_robot_startup"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("sigma_x", double_t, 0, "sigma of distribution on x velocity", 0.3, 0.0001, 10.0)
gen.add("sigma_y", double_t, 0, "sigma of distribution on y velocity", 0.3, 0.0001, 10.0)
gen.add("sigma_z", double_t, 0, "sigma of distribution on z velocity", 0.3, 0.0001, 10.0)
gen.add("sigma_roll", double_t, 0, "sigma of distribution on roll angular velocity", 0.3, 0.0001, 10.0)
gen.add("sigma_pitch", double_t, 0, "sigma of distribution on pitch angular velocity", 0.3, 0.0001, 10.0)
gen.add("sigma_yaw", double_t, 0, "sigma of distribution on yaw angular velocity", 0.3, 0.0001, 10.0)
gen.add("feedback_enabled_sigma", double_t, 0, "Odometry feedback is enabled when sigma is larger than this value", 0.0001, 0.3, 10.0)

exit(gen.generate(PACKAGE, "odometry_feedback_wrapper_reconfigure", "OdometryFeedbackWrapperReconfigure"))
